cron/optc.php

```diff
@@ -77,7 +84,8 @@
 foreach ($users as $username => $data) {
     $jsonData["members"][] = [
         "username" => $username,
-        "count" => $data['imageCount']
+        "count" => $data['imageCount'],
```

Comment:
Total count would need to replace imageCount here. Output json needs to be the same format.